modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFClaim.testcase

```diff
@@ -206,6 +206,226 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-178119. Verify that is possible to export all visible Claims"
+	@priority = 4
+	test CanExportFilteredClaims {
+		task ("Given a new account is created") {
+			JSONAccountEntry.addAccountEntry(
+				accountEntryName = "Company Name 2",
+				accountEntryType = "Business");
+
+			JSONAccountEntryUser.addExistUsertoAccount(
+				accountEntryName = "Company Name 2",
+				userEmailAddress = "partner@liferay.com");
+
+			JSONAccountEntryUser.addExistUsertoAccount(
+				accountEntryName = "Company Name 2",
+				userEmailAddress = "manager@liferay.com");
+		}
+
+		task ("And a new MDF is created with a different account") {
+			JSONPRM.createMDFRequest(
+				accountEntryName = "Company Name 2",
+				activityName = "Activity Name 3",
+				campaignName = "Campaign Name 1",
+				secondActivityName = "Activity Name 4",
+				secondBudgetValueList = 1000);
+		}
+
+		task ("And that the Manager user is logged in and goes to MDF listing page") {
```

Comment:
Hi @marinaasant0s, this 'that' in our task isn't necessary. So it should be "And the Manager user is logged in and goes to MDF listing page".